subsys/bluetooth/Kconfig.iso

```diff
@@ -11,13 +11,15 @@ config BT_ISO
 config BT_ISO_UNICAST
 	bool
 	depends on BT_CONN
+	depends on !BT_CTLR || BT_CTLR_CENTRAL_ISO_SUPPORT || BT_CTLR_PERIPHERAL_ISO_SUPPORT
```

Comment:
This line shouldn't be necessary as `BT_ISO_UNICAST` is a hidden config enabled by `BT_ISO_PERIPHERAL` or `BT_ISO_CENTRAL` where you've added the same dependencies